test/expected/edition.out

```diff
@@ -14,63 +32,39 @@ SELECT _timescaledb_internal.tsl_loaded();
  t
 (1 row)
 
-SELECT _timescaledb_internal.enterprise_enabled();
- enterprise_enabled 
---------------------
- f
-(1 row)
-
-\unset ECHO
-SET timescaledb.license_key='ApacheOnly';
-ERROR:  invalid value for parameter "timescaledb.license_key": "ApacheOnly"
-DETAIL:  Cannot downgrade a running session to Apache Only.
-HINT:  change the license in the configuration file
+SET timescaledb.license='apache';
+ERROR:  invalid value for parameter "timescaledb.license": "apache"
+DETAIL:  Cannot downgrade a running session to Apache license.
+HINT:  Change the license in the configuration file.
 SELECT allow_downgrade_to_apache();
  allow_downgrade_to_apache 
 ---------------------------
  
 (1 row)
 
-SET timescaledb.license_key='ApacheOnly';
-select * from timescaledb_information.license;
- edition | expired | expiration_time 
----------+---------+-----------------
- apache  | f       | infinity
-(1 row)
-
-SELECT _timescaledb_internal.current_db_set_license_key('CommunityLicense');
- current_db_set_license_key 
-----------------------------
- CommunityLicense
+SET timescaledb.license='apache';
+SHOW timescaledb.license;
+ timescaledb.license 
+---------------------
+ apache
 (1 row)
 
-select * from timescaledb_information.license;
-  edition  | expired | expiration_time 
------------+---------+-----------------
- community | f       | infinity
+SELECT set_license('tsl');
```

Comment:
What is this function testing, as opposed to just `SET timescaledb.license =`?

Looks like it is testing database-level settings and session-level settings at once, but I don't get what I should expect to happen. Seems it just sets the license and it works.

I guess a more interesting use case is to have only database-level setting so that is the source of the variable.